httpcore/_async/connection.py

```diff
@@ -23,11 +23,13 @@ def __init__(
         http2: bool = False,
         ssl_context: SSLContext = None,
         socket: AsyncSocketStream = None,
+        local_addr: bytes = None,
```

Comment:
Just to check - Do we want `local_addr` or `source_addr` here?
What are trio, asyncio, and the sync stdlib using for their naming?

Reply:
asyncio uses `local_addr`, trio uses `local_address`, and the stdlib socket module uses `source_address` (and curio uses `source_addr`).  There's really no consistency.